api/google_api/routes/Calendar.js

```diff
@@ -25,4 +25,22 @@ router.get('/getCalendarEvents', async (req, res) => {
     });
 });
 
+router.post('/addEventToCalendar', async (req, res) => {
```

Comment:
add to [`test/Calendar.js`](https://github.com/SCE-Development/Core-v4/tree/dev/test/Calendar.js) so we can test this behavior out too